cypress/support/commands.js

```diff
@@ -28,14 +28,74 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 
 addMatchImageSnapshotCommand();
 
-Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+Cypress.Commands.add('getByDataQa', (selector) => {
+  cy.get(`[data-qa="${selector}"]`);
```

Comment:
The task description requires you to add `data-qa` attributes to elements, but you replaced `data-cy` with `data-qa` instead of adding `data-qa`. You need to both add `data-qa` and keep `data-cy`